### PR TITLE
SorrTask770 Removed the global variable

### PR DESCRIPTION
**Pull request by Vedanshu7**
#770 

- Removed the global variable and related implementation.

### CODE REVIEW

**Review by sonaalKant — COMMENTED** [review]
The purpose of using `_NEXT_ORDER_SIDE_PER_ASSET` is that, we don't always buy or always sell for any particular symbol. Removing this will leave this on chance. 

```
        if order_direction == "sell" or (
            order_direction is None and np.random.choice([True, False])
        ):
```
My opinion: for longer runs the probability will be closer to 50% and for short runs it does not matter as we flatten anyways.

wdyt? @jsmerix Is it fine to remove this altogether or should we have two different modes : `random` and `random_with_balance`

**Comment by DanilYachmenev** [comment]
@sonaalKant what status with this one?

**Comment by jsmerix** [comment]
> wdyt? @jsmerix Is it fine to remove this altogether or should we have two different modes : random and random_with_balance

I think we should keep the feature to balance the positions - for larger universe it might come in handy because even a slight disbalance can block you from placing orders further.

File: oms/broker/ccxt/scripts/run_ccxt_broker.py [hunk 1]
```diff
@@ -226,18 +241,22 @@ def _get_random_orders(
...
+    order_sides = {symbol: None for symbol in symbols}
     orders = []
     for asset_id in asset_ids:
         position = positions.get(asset_id["asset_id"], 0)
         if close_positions and position == 0:
             continue
         # Set order direction for the current order.
         symbol = asset_id["symbol"]
-        _LOG.debug(_NEXT_ORDER_SIDE_PER_ASSET)
+        _LOG.debug(order_sides)
```
Comment (gpsaggese) [inline oms/broker/ccxt/scripts/run_ccxt_broker.py:255]: It should be ("%s", order_sides)

**Review by gpsaggese — COMMENTED** [review]
LGTM

**Comment by sonaalKant** [comment]
LGTM. I will add broker team to know of these changes. @samarth9008 @Sameep2808

**Review by sonaalKant — APPROVED** [review]

**Review by samarth9008 — COMMENTED** [review]
The script LGTM

Did we check if the script is working fine?

If not, lets move this code to cmamp, close this PR. On cmamp we already have a smoke test which can verfiy for us.

**Comment by Vedanshu7** [comment]
> The script LGTM
> 
> Did we check if the script is working fine?
> 
> If not, lets move this code to cmamp, close this PR. On cmamp we already have a smoke test which can verfiy for us.

https://github.com/cryptokaizen/cmamp/pull/9055

**Comment by samarth9008** [comment]
Closing as moved to cmamp

**Pull request closed without merging**